apps/explorer/src/utils/format.ts

```diff
@@ -245,7 +245,7 @@ export function formatCalculatedPriceToDisplay(
   const buySymbol = safeTokenName(buyToken)
   const sellSymbol = safeTokenName(sellToken)
 
-  const [quoteSymbol] = isPriceInverted ? [sellSymbol, buySymbol] : [buySymbol, sellSymbol]
+  const [quoteSymbol] = isPriceInverted ? [buySymbol, sellSymbol] : [sellSymbol, buySymbol]
```

Comment:
Nitpick, which is definitely not coming from this pr so feel free to ignore:

Since base symbol is not used, just return the quote and get rid of the array

```suggestion
  const quoteSymbol = isPriceInverted ? buySymbol : sellSymbol
```

Reply:
Yep! good point.